### ISSUE TITLE
tcl: fatal error: stdlib.h: No such file or directory

### ISSUE THREAD

**Issue by tom-wa** [ElektraInitiative/libelektra#949]
```
/usr/include/c++/6/cstdlib:75:25: fatal error: stdlib.h: No such file or directory
 #include_next <stdlib.h>
```

**Comment by markus2330** [comment]
Seems to be a regression because of the -isystem?

**Comment by markus2330** [comment]
It is a gcc bug: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=70129

Seems like have an explicit check if INCLUDE_SYSTEM_DIRECTORY is a default directory is necessary....

**Comment by markus2330** [comment]
fixed in 3822b50d1256e996a2adce694630ab3e66177037
